src/css/_higlass.scss

```diff
@@ -0,0 +1,48 @@
+@import "~higlass/dist/hglib.css";
+
+@mixin higlass($theme-name, $theme-colors) {
+    .v-higlass-title-wrapper {
+        height: calc(100% - 20px);
+        .card-body {
+          width: inherit;
+          height: inherit;
+          padding: 5px;
+          background-color: map-get($theme-colors, "secondary-background");
+          .v-higlass-wrapper-parent {
+            width: inherit;
+            height: inherit;
+            display: block;
+            position: relative;
+            box-sizing: border-box;
+            font-size: 12px;
+            line-height: 1.42857143;
```

Comment:
Curious where this came from?

Reply:
Probably this `.btn` style in the higlass CSS: https://github.com/higlass/higlass/blob/575f71f7e96f966eb7672daf802fa4bc92501756/app/styles/GenomePositionSearchBox.module.scss#L109

Regardless, I just removed it and everything still looks fine.